site/content/docs/integrations/terraform.md

```diff
@@ -24,22 +24,38 @@ To get started, install the latest version of the Checkly Terraform provider fro
 Your `main.tf` file will look similar to the following:
 
 ```terraform
-variable "checkly_api_key" {}
-
 terraform {
   required_providers {
     checkly = {
       source = "checkly/checkly"
-      version = "1.2.0"
+      version = "1.3.0"
     }
   }
 }
 
+variable "checkly_api_key" {}
+variable "checkly_account_id" {}
+
 provider "checkly" {
   api_key = var.checkly_api_key
+  account_id = var.checkly_account_id
 }
 ```
 
+To use the provider with your Checkly account, you will need an API Key for your Checkly user. Go to the [User Settings: API Keys page](https://app.checklyhq.com/settings/user/api-keys) and click 'Create API Key'. Get your User API Key and add it to your env:
+```bash
+$ export TF_VAR_checkly_api_key=cu_xxx
+```
+
+1. You also need to set your target Account ID, you can find the Checkly Account ID under your [account settings](https://app.checklyhq.com/settings/account/general). If you don't have access to account settings, please contact your account owner/admin.
```

Comment:
@ianaya89 typo? no point in having a list with only one element.

Reply:
Also, I would just say: "You also need to set your target Account ID, which you can find under your account settings. "